docs/Configuration.rst

```diff
@@ -0,0 +1,93 @@
+Configuration
+=============
+The recommend method to override the configuration file is by letting
+lago auto-generate them::
```

Comment:
Actually it puts the commented out version in ``/etc/lago/lago.conf`` exactly for that(reference purposes), so maybe I should add that to the documentation. I think its better to output the actual loaded configuration as it might be useful for debugging also(i.e. seeing what is the current configuration for a parameter if you are not sure what is loaded) and not only for editing, in that sense maybe the verb name is a bit misleading(generate-config vs something like show-config). Once someone already edits the config file I assume he will remove what he finds redundant.

As for the documentation, maybe change it to something like:
>The /etc/lago/lago.conf file is commented out and used as an example. You can edit it there and change the global configuration for all users. Also, you may dump the current loaded configuration using ``lago generate-config`` and edit what you need, like this:'

and then the example in the current PR. 

as a side note, another thought I had was to add something like ``lago config`` which will be local to the current used prefix(like ``git config``), this could really simplify stuff for the user(one none-trivial problem is that the prefix exists only after ``lago init``, other than that I think it should be simple)